plugin/src/test/groovy/com/novoda/buildproperties/BuildPropertiesTest.groovy

```diff
@@ -166,6 +166,30 @@ class BuildPropertiesTest {
         assertThat(project.buildProperties.chain['f']).hasValue('value_f')
     }
 
+    @Test
+    public void shouldReturnTrueWhenBuildPropertiesContainKey() {
+        environment.set('a', 'value_a')
+
+        project.buildProperties {
+            env {
+                using System.getenv()
+            }
+        }
+
+        assertThat(project.buildProperties.env.contains('a')).isTrue()
+    }
```

Comment:
@tobiasheine does it make sense to add a test that validates that property 'a' value is actually 'value_a'  ?

Reply:
We have these kind of asserts already in other test cases, for example:
`assertThat(project.buildProperties.map['a']).hasValue('value_a')`